src/cmds/qmgr.c

```diff
@@ -2005,6 +2005,11 @@ display(int otype, int ptype, char *oname, struct batch_status *status,
 				printf("Server %s\n", status->name);
 		}
 		else if (otype == MGR_OBJ_SCHED) {
+			if(oname && *oname && strncmp(oname,status->name,PBS_MAXSCHEDNAME)) {
+				status = status->next;
```

Comment:
Can you please add a comment here explaining the logic? why is 'status' being moved to the sched object in the list?

Reply:
### Logic: 
Since now pbs_statsched(0 dumps batch_status for all the scheduler, we need to skip over the one we are not interested in. Hence advancing the status.

Reply:
Ok. I was a bit confused by the fact that none of the other objects do this so I looked a bit into how it's used. It seems like 'oname' is used by display just to ensure that only custom resources are displayed when somebody types "list resource" without specifying an object name. It seems like a very specific use case for just making sure that built-in resources don't get displayed. I don't think you need to bother with it at all for the sched object, the results are already filtered by the pbs_stat calls, display just needs to display them and not do any filtering itself, resources are an exception.

Reply:
disagree, justified above